### PR TITLE
Add New Post -  pritambera2000

### PR DESCRIPTION
**Pull request by pritambera2000**
let me know if there are any issues

### CODE REVIEW

**Comment by tasnimzotder** [comment]
hey @pritambera2000, I forgot to mention that the images should be in `public/assets/md-images/` directory. So, could you please move the images to that directory?

File: md-posts/pritambera2000.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+cover_img: 'https://assets.nrdc.org/sites/default/files/styles/header_background/public/personalaction-kidsclimatechange-1200x630.jpg?itok=JBsv97mu'
+excerpt: 'Climate change & Food hunger are the greatest problems of the century, human activities causing problems like pollution,global warming which are leading to climate change and upsetting the balance of the earth.'
+---
+
+## What is Climate change?
+Climate is the long-term pattern of weather in a particular area. People often confuse weather and climate; the difference is really a matter of time. Climate change refers to long-term shifts in temperatures and weather patterns. These shifts may be natural, such as through variations in the solar cycle. But since the 1800s, human activities have been the main driver of climate change, primarily due to burning fossil fuels like coal, oil and gas.
+
+  > “Climate change is real. It is happening right now, it is the most urgent threat facing our entire species and we need to work collectively together and stop procrastinating.”  --Leonardo Di Caprio, Actor & Environmentalist
+
+<img src="../assets/md-images/ClimateChange.jpg" max-height="200px"  max-width="300px" >
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:14]:
```suggestion
<img src="/assets/md-images/ClimateChange.jpg" max-height="200px"  max-width="300px" >
```

File: md-posts/pritambera2000.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+## Causes of climate change
+There are many reasons of climate change but Scientists telling that the recent climate change is largely caused by human activities. Here are some key point-
+- **Deforestation** , because living trees absorb and store carbon dioxide
+- **Increasingly intensive agriculture**  which emits greenhouse gases like methane and nitrous oxide
+- **Fluorinated gases** are emitted from equipment and products that use these gases. Such emissions have a very strong warming effect, up to 23 000 times greater than CO2.
+<img src="../assets/md-images/Industry.jpeg" height="auto" width="100%" max-width="400px" >
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:21]:
```suggestion
<img src="/assets/md-images/Industry.jpeg" height="auto" width="100%" max-width="400px" >
```

File: md-posts/pritambera2000.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+- **Greenhouse gases** and the greenhouse effect – Some gases in the Earth's atmosphere trap heat and stop it escaping into space. We call these 'greenhouse gases'. These gases act as a warming blanket around the Earth, known as the 'greenhouse effect'. Greenhouse gases come from both human and natural sources. Gases like carbon dioxide, methane, and nitrous oxide naturally occur in the atmosphere. Others, such as chlorofluorocarbons (CFCs), are only produced by human activity.
+<img src="../assets/md-images/GreenHouse.jpeg" height="100%" width="100%" max-width="300px" >
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:23]:
```suggestion
<img src="/assets/md-images/GreenHouse.jpeg" height="100%" width="100%" max-width="300px" >
```

File: md-posts/pritambera2000.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+## What is the issue?
+Humans and wild animals face new challenges for survival because of climate change. More frequent and intense drought, storms, heat waves, rising sea levels, melting glaciers and warming oceans can directly harm animals, destroy the places they live, and wreak havoc on people’s livelihoods and communities
+
+- Effects on species  Almost every species facing direct and indirect bad effects of climate change. But some  Atlantic  species are highly endengered because of climate change especially global Warming
+  - **Polar Bear** The Arctic warming about twice as fast as the global average, causing the ice that polar bears  depend on to melt away.Loss of sea ice, polar bears are forced to stay longer on land.This is increasingly bringing them into contact with local people in Antarctic villages and towns , leading to bears and people being injured or killed in self defense.
+<img src="../assets/md-images/PolarBear.jpeg" max-height="50px" width="100%" max-width="300px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:30]:
```suggestion
<img src="/assets/md-images/PolarBear.jpeg" max-height="50px" width="100%" max-width="300px">
```

File: md-posts/pritambera2000.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  
+  - **Snow leopards** Warming in the Himalayas has already occurred at three times the global average. This is prime snow leopard habitat and continued warming will cause their range to shrink as the treeline moves higher up the mountains. This will not only fragment and isolate snow leopard populations, but it will severely affect their prey too.
+ 
+  - **Adelie Penguins** In parts of the Antarctic peninsula, sea ice cover has reduced by over 60% in 30 years. One of the Adélies’ main food sources, krill, needs sea ice to breed and feed under, and the loss of ice is thought to be causing a decline in krill in some areas. Warmer temperatures can also affect Adélie chick hatching times, and they may hatch at times when food is more scarce.
+<img src="../assets/md-images/penguine.jpg" height="0%" width="100%" max-width="300px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:36]:
```suggestion
<img src="/assets/md-images/penguine.jpg" height="0%" width="100%" max-width="300px">
```

File: md-posts/pritambera2000.md [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+4.  **Educate Yourself and Others** The importance of educating others about climate change cannot be overstated in our modern society. There are many platforms for us to utilize that can allow us to spread our message easily. Whether you use word of mouth or social media, there are always ways to educate others on what climate change is doing to our planet. You can help protect the planet by educating others about the dangers of climate change and how to act against it.
+
+## World Hunger 
+> Hunger defines a short-term physical discomfort as a result of chronic food shortage, or in severe cases, a life-threatening lack of food. (National Research Council, 2006)
+> 
+> World hunger refers to hunger aggregated to the global level. Related terms include food insecurity and malnutrition --worldhumger.org
+
+<img src="../assets/md-images/WorldHunger2.jpeg" height="auto" width="100%" max-width="300px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:54]:
```suggestion
<img src="/assets/md-images/WorldHunger2.jpeg" height="auto" width="100%" max-width="300px">
```

File: md-posts/pritambera2000.md [hunk 7]
```diff
@@ -0,0 +1,74 @@
...
+## Climate change and World Hunger
+
+Climate change and World hunger are closely related to each other.The wired behavior of climate due to climate change plays a vital role in agriculture that's why the production of food is affected and causes world hunger..
+### Climates role in World Hunger
+1. **Impact in Food production**  Climate change decreases the level of agricultural productivity, which has a direct impact on hunger levels globally.
+2. **Impact in Nutrition** Climate variability and extremes can also affect nutrition and food safety in several ways.
+3. **Impact on food access**  Extreme climate change effect the prices of food . so the rural poor peoples can't afford food
+
+
+<img src="../assets/md-images/WorldHunger1.jpeg" height="auto" width="100%" max-width="700px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:65]:
```suggestion
<img src="/assets/md-images/WorldHunger1.jpeg" height="auto" width="100%" max-width="700px">
```

**Comment by pritambera2000** [comment]
@tasnimzotder  ok , i moved images please check..

File: md-posts/pritambera2000.md [hunk 8]
```diff
@@ -0,0 +1,74 @@
...
+## Climate change and World Hunger
+
+Climate change and World hunger are closely related to each other.The wired behavior of climate due to climate change plays a vital role in agriculture that's why the production of food is affected and causes world hunger..
+### Climates role in World Hunger
+1. **Impact in Food production**  Climate change decreases the level of agricultural productivity, which has a direct impact on hunger levels globally.
+2. **Impact in Nutrition** Climate variability and extremes can also affect nutrition and food safety in several ways.
+3. **Impact on food access**  Extreme climate change effect the prices of food . so the rural poor peoples can't afford food
+
+
+<img src="../public/assets/md-images/WorldHunger1.jpeg" height="auto" width="100%" max-width="200px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:65]:
```suggestion
<img src="/assets/md-images/WorldHunger1.jpeg" height="auto" width="100%" max-width="200px">
```

File: md-posts/pritambera2000.md [hunk 9]
```diff
@@ -0,0 +1,74 @@
...
+4.  **Educate Yourself and Others** The importance of educating others about climate change cannot be overstated in our modern society. There are many platforms for us to utilize that can allow us to spread our message easily. Whether you use word of mouth or social media, there are always ways to educate others on what climate change is doing to our planet. You can help protect the planet by educating others about the dangers of climate change and how to act against it.
+
+## World Hunger 
+> Hunger defines a short-term physical discomfort as a result of chronic food shortage, or in severe cases, a life-threatening lack of food. (National Research Council, 2006)
+> 
+> World hunger refers to hunger aggregated to the global level. Related terms include food insecurity and malnutrition --worldhumger.org
+
+<img src="../public/assets/md-images/WorldHunger2.jpeg" height="auto" width="100%" max-width="200px">
```
Comment (tasnimzotder) [inline md-posts/pritambera2000.md:54]:
```suggestion
<img src="/assets/md-images/WorldHunger2.jpeg" height="auto" width="100%" max-width="200px">
```

**Comment by pritambera2000** [comment]
Hey ,@tasnimzotder images are over zooming in the full page can you help me in resizing the images properly. i'm newbie in markdown :(

**Comment by tasnimzotder** [comment]
in `pritambera2000.md` only file, change `../public/assets/md-images/*` to `/assets/md-images/*`

**Comment by tasnimzotder** [comment]
join discord -> https://discord.com/invite/f4sYVwS8C6

**Comment by pritambera2000** [comment]
> in `pritambera2000.md` only file, change `../public/assets/md-images/*` to `/assets/md-images/*`

changed..

**Review by tasnimzotder — APPROVED** [review]
It's working fine 🎉

**Comment by tasnimzotder** [comment]
Thanks @pritambera2000, for your contributions :)

**Pull request merged**